common/app/mvt/MultiVariateTesting.scala

```diff
@@ -68,8 +68,29 @@ object SeriesOnwardPosition extends TestDefinition(
     }
 }
 
+object IntersperseMultipleStoryPackagesStories extends TestDefinition(
+  List(Variant0, Variant1, Variant2, Variant3, Variant4), // 5% of our audience
+  "intersperse-multiple-story-packages-stories",
+  "To test if mixing storyPackages stories (when article has more than one storyPackage) results in more clicks",
+  new LocalDate(2016, 5, 3)
+)
+object IntersperseMultipleStoryPackagesStoriesControl extends TestDefinition(
+  List(Variant5, Variant6, Variant7, Variant8, Variant9), // 5% of our audience
+  "intersperse-multiple-story-packages-stories-control",
+  "Control group for intersperse-multiple-story-packages-stories A/B test",
+  new LocalDate(2016, 5, 3)
+)
+
```

Comment:
I guess you could pick variants which are not being used, but that might reduce your audience size. How big does the test need to be?